### ISSUE TITLE
SUEWS 2018b - Observed Qf (emissionsmethod = 0) did not succeed

### ISSUE THREAD

**Issue by sunt05** [UMEP-dev/UMEP#79]
**[Original report](https://bitbucket.org/fredrik_ucg/umep/issue/79) by rary2013 (Bitbucket: [rary2013](https://bitbucket.org/rary2013), ).**

----------------------------------------

From the SUEWS Manual: " if available, the anthropogenic heat ﬂux can be provided in the met forcing ﬁle (and set EmissionsMethod = 0 in RunControl.nml), in which case all columns here except Code and BaseTHDD should be set to ’-999’."

This did not work. 
i.e. Simulation with emissionsmethod = 0 did not succeed.

Program stopped! Profile code   -999.00000000000000      not found in SUEWS_Profiles.txt.
 >>> See problems.txt for serious issues in the run <<<
 ERROR! SUEWS run stopped.
 problem: Cannot find code in SUEWS_Profiles.txt

Solution:
Setting all columns other than Code and BaseTHDD to "99999" worked.

In summary (as an example):

Create new Code 670 in SUEWS_AnthropogenicHeat.txt, with BaseTHDD = 18.2 and all other columns 99999.

In SUEWS_Profiles.txt make sure Code 99999 exists and all its other columns set to -999

In SUEWS_SiteSelect.txt, make sure AnthropogenicCode = 670
